ui/imports/shared/stores/CommunityTokensStore.qml

```diff
@@ -34,7 +34,7 @@ QtObject {
         communityTokensModuleInst.deployCollectible(communityId, collectibleItem.accountAddress, collectibleItem.name,
                                                     collectibleItem.symbol, collectibleItem.description, collectibleItem.supply,
                                                     collectibleItem.infiniteSupply, collectibleItem.transferable, collectibleItem.remotelyDestruct,
-                                                    collectibleItem.chainId, collectibleItem.artworkSource/*instead: jsonArtworkFile*/)
+                                                    collectibleItem.chainId, collectibleItem.artworkSource, jsonArtworkFile)
```

Comment:
I think you can remove collectibleItem.artworkSource. jsonArtworkFile contains imagePath.